modules/images-create-s2i-scripts.adoc

```diff
@@ -25,47 +24,42 @@ script specified in a `BuildConfig` can take one of the following forms:
 |Script |Description
 
 |`assemble`
-(required)
-|The `assemble` script builds the application artifacts from a source and places them into appropriate directories inside the image. The workflow for this script is:
+|The `assemble` script builds the application artifacts from a source and places them into appropriate directories inside the image. This script is required. The workflow for this script is:
 
-. Restore build artifacts. If you want to support incremental builds, make sure to define *_save-artifacts_* as well (optional).
+. Restore build artifacts. If you want to support incremental builds, make sure to define `save-artifacts` as well. This is optional.
 . Place the application source in the desired location.
 . Build the application artifacts.
 . Install the artifacts into locations appropriate for them to run.
 
 |`run`
-(required)
-|The `run` script executes your application.
+|The `run` script executes your application. This script is required.
 
 |`save-artifacts`
-(optional)
-|The `save-artifacts` script gathers all dependencies that can speed up the build processes that follow. For example:
+|The `save-artifacts` script gathers all dependencies that can speed up the build processes that follow. This script is optional. For example:
 
-- For Ruby, `gems` installed by Bundler.
-- For Java, `.m2` contents.
+* For Ruby, `gems` installed by Bundler.
+* For Java, `.m2` contents.
 
-These dependencies are gathered into a tar file and streamed to the standard output.
+These dependencies are gathered into a `tar` file and streamed to the standard output.
 
 |`usage`
-(optional)
-|The `usage` script allows you to inform the user how to properly use your image.
+|The `usage` script allows you to inform the user how to properly use your image. This script is optional.
 
 |`test/run`
-(optional)
-|The `test/run` script allows you to create a simple process to check if the image is working correctly. The proposed flow of that process is:
+|The `test/run` script allows you to create a process to check if the image is working correctly. This script is optional. The proposed flow of that process is:
 
 . Build the image.
 . Run the image to verify the `usage` script.
 . Run `s2i build` to verify the `assemble` script.
-. Run `s2i build` again to verify the `save-artifacts` and `assemble` scripts save and restore artifacts functionality. (optional)
+. Run `s2i build` again to verify the `save-artifacts` and `assemble` scripts save and restore artifacts functionality. This is optional.
```

Comment:
Would it be better as "Optional: ....." ? It would inform the user it's optional before they read the entire sentence and give them the option to glance over it?

Reply:
Yes! Thank you.